swap/swap.go

```diff
@@ -127,12 +127,11 @@ func keyToID(key string, prefix string) enode.ID {
 }
 
 // createOwner assings keys and addresses
-func createOwner(prvkey *ecdsa.PrivateKey, contract common.Address) *Owner {
+func createOwner(prvkey *ecdsa.PrivateKey) *Owner {
 	pubkey := &prvkey.PublicKey
 	return &Owner{
```

Comment:
since we are already making changes to this function: could we go ahead and instantiate this `Owner` struct with its fields ordered in the same way as it is declared?

Reply:
Will do!